guides/common/assembly_using-pxe-to-provision-hosts.adoc

```diff
@@ -21,6 +20,7 @@ ifndef::satellite[]
 :parent-client-pkg-ext: {client-pkg-ext}
 :secureboot-os-name: My_Operating_System_In_Lowercase
```

Comment:
@maximiliankolb There actually is an attribute for that but it isn't being modified anywhere :laughing: 
Perhaps it's a good idea to request a PR to do that for the other OSs.
I'm not sure what the value is supposed to be, e.g. is `rhel` good enough? Does the directory name even matter as long as it's consistent in the whole procedure?

Reply:
Yeah, let's keep it consistent for now, and then think/ask if we can overwrite this attribute for each OS in a follow-up PR.